builder.go

```diff
@@ -132,8 +116,7 @@ func (b *Bob) maybeQueue(path string) {
 		// we couldn't find the file
 		// most likely a deletion
 		delete(mtimes, path)
-		b.execute()
-		return
+		return true
 	}
 
 	mtime := stat.ModTime()
```

Comment:
We still get duplicate events for Remove/Create. It's the same issue we had with the other package. It's a common thing for OSX to send indexing events and duplicate events from what I've seen.